extensions/auth/portability-auth-microsoft/src/main/java/org/dataportabilityproject/auth/microsoft/MicrosoftAuthDataGenerator.java

```diff
@@ -51,6 +52,7 @@
           .put("mail", ImmutableList.of("user.read", "Mail.Read"))
           .put("contacts", ImmutableList.of("user.read", "Contacts.Read"))
           .put("calendar", ImmutableList.of("user.read", "Calendars.Read"))
+          .put("derived-data", ImmutableList.of("user.read", "Files.Read.All"))
```

Comment:
The Files.Read.All isn't obvious to me, can you document why that scope is used for derived data?

Reply:
Sure